ReleaseChecklist.md

```diff
@@ -20,50 +20,52 @@
+ - [ ] Run ``./update --reuse-hashes`` in ``solc-bin`` and verify that the script has updated ``list.js``, ``list.txt`` and ``list.json`` files correctly and that symlinks to the new release have been added in ``solc-bin/wasm/`` and ``solc-bin/emscripten-wasm32/``.
+ - [ ] Create a pull request and merge.
 
 ### Homebrew and MacOS
  - [ ] Update the version and the hash (``sha256sum solidity_x.x.x.tar.gz``) in https://github.com/Homebrew/homebrew-core/blob/master/Formula/solidity.rb
  - [ ] Update the version and the hash (``sha256sum solidity_x.x.x.tar.gz``) in https://github.com/ethereum/homebrew-ethereum/blob/master/solidity.rb
- - [ ] Take the binary from the ``b_osx`` run of the released commit in circle-ci and add it to the release page as ``solc-macos``.
 
-### Update solc-bin
- - [ ] Copy ``soljson.js`` from the release page to ``solc-bin/bin/soljson-v<version>+commit.<commit>.js``
- - [ ] Copy ``solc-static-linux`` from the release page to ``solc-bin/linux-amd64/solc-linux-amd64-v<version>+commit.<commit>``
- - [ ] Copy ``solc-macos`` from the release page to ``solc-bin/macos-amd64/solc-macos-amd64-v<version>+commit.<commit>``
- - [ ] Copy ``solc-windows.zip`` from the release page to ``solc-bin/windows-amd64/solc-windows-amd64-v<version>+commit.<commit>.zip``
- - [ ] Make the linux and the macos binaries executable.
- - [ ] Run ``./update`` in ``solc-bin`` and verify that the script has updated ``list.js``, ``list.txt`` and ``list.json`` files correctly and that symlinks to the new release have been added in ``solc-bin/wasm/`` and ``solc-bin/emscripten-wasm32/``.
- - [ ] Create a pull request and merge.
+### Docker
+ - [ ] Run ``./scripts/docker_deploy_manual.sh v$VERSION``).
 
 ### PPA
  - [ ] Change ``scripts/release_ppa.sh`` to match your key's email and key id.
- - [ ] Run ``scripts/release_ppa.sh release`` to create the PPA release (you need the relevant openssl key).
+ - [ ] Run ``scripts/release_ppa.sh v$VERSION`` to create the PPA release (you need the relevant openssl key).
  - [ ] Wait for the ``~ethereum/ubuntu/ethereum-static`` PPA build to be finished and published for *all platforms*. SERIOUSLY: DO NOT PROCEED EARLIER!!! *After* the static builds are *published*, copy the static package to the ``~ethereum/ubuntu/ethereum`` PPA for the destination series ``Trusty`` and ``Xenial`` while selecting ``Copy existing binaries``.
 
-### Docker
- - [ ] Check that the Docker release was pushed to Docker Hub (this still seems to have problems, run ``./scripts/docker_deploy_manual.sh v0.x.x``).
-
 ### Documentation
  - [ ] Build the new version on https://readthedocs.org/projects/solidity/ (select `latest` on the bottom of the page and click `BUILD`)
  - [ ] In the admin panel, select `Versions` in the menu and set the default version to the released one.
 
 ### Release solc-js
+ - [ ] Wait until solc-bin was properly deployed. You can test this via remix - a test run through remix is advisable anyway.
```

Comment:
You mean until the files become available at `https://solc-bin.ethereum.org`?
Then maybe visiting `https://solc-bin.ethereum.org/bin/soljson-v$VERSION+commit.$COMMIT.js` as a sanity check is also a good idea.

Also, this might be a bit too specific to add to the list but making sure that the [S3 sync action](https://github.com/ethereum/solc-bin/actions?query=workflow%3A"Mirror+repository+content+to+an+S3+bucket") did not fail might be a good idea (or at least it might give some feedback on the progress of the sync).

Reply:
I think the test via remix is better. You can also do a simple compilation run and retrieve the version reported by the compiler.

Reply:
Sure. Just a suggestion.